Resources/Prototypes/Entities/Clothing/Uniforms/jumpsuits.yml

```diff
@@ -1,3 +1,21 @@
+- type: entity
+  parent: ClothingUniformBase
+  id: ClothingUniformJumpsuitDeadSquad
+  name: Dead Squad uniform
+  description: Advanced armored jumpsuit used by special forces in special operations.
+  components:
+  - type: Sprite
+    sprite: Clothing/Uniforms/Jumpsuit/deadsquad.rsi
+  - type: Clothing
+    sprite: Clothing/Uniforms/Jumpsuit/deadsquad.rsi
+  - type: Armor
+    modifiers:
+      coefficients:
+        Blunt: 0.8
+        Slash: 0.8
+        Piercing: 0.8
+        Heat: 0.8
```

Comment:
Jumpsuits shouldn't have armour.

Reply:
ERT have, why should not death squad ?

Reply:
ERT has armor, but the squadron shouldn't?  this is weird.  From the point of view of balance, the squadron should not die at all, and even these 20% will not change anything

Reply:
also hos turtleneck has a bit of armour too